db/migrate/20170821181415_create_conditions.rb

```diff
@@ -1,14 +1,15 @@
 class CreateConditions < ActiveRecord::Migration[5.1]
   def change
     create_table :conditions do |t|
-      t.date    :date
+      t.date  :weather_date
```

Comment:
Think you could've stayed with date here? No need to change it back, but intrigued to hear why you changed it.